doc/self-hosted/index.md

```diff
@@ -50,10 +50,27 @@ Options:
   -p, --port <number>         Port to bind on. (default: 8443)
   -N, --no-auth               Start without requiring authentication.
   -H, --allow-http            Allow http connections.
-  -P, --password <value>      Specify a password for authentication.
+  -P, --password <value>      DEPRECATED in 1.903-vsc1.33.1: Use "PASSWORD" environment variable instead. Specify a password for authentication. 
   --disable-telemetry         Disables ALL telemetry.
   --help                      output usage information
   ```
+  
+  ### Password
+  If you're using the binary, the latest version of code-server allows you to set an environment variable PASSWORD
+
+```
+export PASSWORD="yourpassword"
+./code-server --allow-http
+```
+If you're using docker, you can use the -e flag to pass environment variables to your container
+
+`docker run -it -p 127.0.0.1:8443:8443 -e PASSWORD='yourpassword' -v "${PWD}:/home/coder/project" codercom/code-server --allow-http`
+
+If you're using versions older than 1.903-vsc1.33.1, you must use the --password argument.
```

Comment:
It feels a bit odd to me to document the password flag since we don't want anyone to use it and we want anyone on older versions to upgrade and use the more secure environment variable. Thoughts?

Reply:
So modify line 69 to not mention the —password flag in favor of saying that the env variable was used after that version?

Reply:
I was thinking something like keeping the first paragraph but modifying the first line to say something about the version.

For example something like:
````markdown
### Password
Since version 1.903-vsc1.33.1, code-server allows you to set an environment variable PASSWORD.

```
export PASSWORD="yourpassword"
./code-server --allow-http
```

If you're using docker, you can use the -e flag to pass environment variables to your container.

`docker run -it -p 127.0.0.1:8443:8443 -e PASSWORD='yourpassword' -v "${PWD}:/home/coder/project" codercom/code-server --allow-http`
````